app/src/main/java/com/example/dynamicdiet/MainActivity.kt

```diff
@@ -64,12 +64,12 @@ class MainActivity : ComponentActivity() {
 fun InputWeight(viewModel: MainViewModel, scope:CoroutineScope, scaffoldState: ScaffoldState) {
     var context = LocalContext.current
     val simpleDateFormat = SimpleDateFormat("MM-dd-yyy");
-    val datetime = simpleDateFormat.format(Date())
+    val dateTime = simpleDateFormat.format(Date())
         Row{
             Column(
                 Modifier
                     .fillMaxSize()
-                    .padding(30.dp), horizontalAlignment = Alignment.CenterHorizontally, verticalArrangement = Arrangement.Center) {
+                    .padding(30.dp), horizontalAlignment = Alignment.CenterHorizontally, verticalArrangement = Arrangement.Bottom) {
```

Comment:
Realign input and submit button to bottom so there is more room for recorded weights.